socialdistribution/settings.py

```diff
@@ -0,0 +1,83 @@
+"""
+Django settings for socialdistribution project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'k-1y7cv9s-8*x^s+kp@ylfj+*akf)&7by&43e%0160p25qy6c3'
+
```

Comment:
You can generate a random SECRET_KEY upon application start like [open stack/horizon](https://github.com/openstack/horizon/blob/master/horizon/utils/secret_key.py) does.

I don't think it really matters for our purposes though.

[This](http://fearofcode.github.io/blog/2013/01/15/how-to-scrub-sensitive-information-from-django-settings-dot-py-files/) is an alright solution as well.